### DISCUSSION TITLE
Revamp the Window Layout creation to be more flexible.

### DISCUSSION THREAD

**Discussion by I3igAl** [WowOpenBox/WowOpenBox#95] [Ideas]
Similar to #52 I suppose. I want one big and three small on my main screen, and another big on my right hand screen. I am accomplishing this right now by restricting the layout to a single screen, setting it for 4 windows, and writing down the window size and location numbers into a word document, then editting the saved profile in Notepad++. this is very tedious to do any time i want to change my layout and have two large windows. I tried using the Resize and Move function on the main WOB menu bt that does not shift the overlay numbers with the window.

I think you could make the layout generator build a subset for each screen, but a more direct and rudimentary option would be to have input fields to type in the size and location directly into the window layout menu. WOB 1, enter size and location coordinates. WOB 2 enter size and location, etc.

This is one layout i manually created. I will three box in the main screen, the left hand screen can be watching the AH or fishing or any other stationary task, and the right hand screen can be crafting or doing a solo dungeon or raid or whatever else. flexibility is the goal.

![image](https://user-images.githubusercontent.com/80173503/110281874-5d2b3e80-7f92-11eb-9438-a27f159692c1.png)

**Comment by mooreatv** [comment]
The size/coord input already exist, in the main window?

But more options could indeed be handy (though you can in theory just resize and drag any window and snap it to some grid or fine adjust with arrow keys)
